Dockerfile-prod

```diff
@@ -40,9 +40,11 @@ COPY --chown=node:node src ./src
 
 # Install dependencies and build application
 # Use frozen lockfile for reproducible builds
-# Install all dependencies (including dev) for build process
-RUN pnpm install --frozen-lockfile --prod && \
+# Install ALL dependencies (including dev) for build process
```

Comment:
These changes step back but is probably caused by the base branch being an earlier version not including #60 
Should disappear when main is merged/rebased in this branch

Reply:
Oh that is good to know, thanks for checking again. Then we'll keep it!